php-classes/Slate/CBL/Demonstrations/DemonstrationSkillsRequestHandler.php

```diff
@@ -29,8 +29,15 @@ public static function handleBrowseRequest($options = [], $conditions = [], $res
                 return static::throwNotFoundError('Student not found');
             }
 
-            if (!$GLOBALS['Session']->hasAccountLevel('Staff') && $Student->ID != $GLOBALS['Session']->PersonID) {
-                return static::throwUnauthorizedError('Only staff may browse others\' records');
+            if (
+                    !$GLOBALS['Session']->hasAccountLevel('Staff') &&
+                    $Student->ID != $GLOBALS['Session']->PersonID &&
+                    !$GuardianRelationship = \Emergence\People\GuardianRelationship::getByWhere([
```

Comment:
@nbey you need to use `getByClass` or add a `Class` condition to this to ensure that only relationships of type `GuardianRelationship` match, `getByWhere` on its own searches the whole table and doesn't filter by class hierarchy